File: GTTS/INL.UserService/Models/IUserProfile_Item.cs
using System;
using System.Collections.Generic;

namespace INL.UserService.Models
{
	public interface IUserProfile_Item
	{
		int AppUserID { get; set; }
		string ADOID { get; set; }
		string First { get; set; }
		string Middle { get; set; }
		string Last { get; set; }
		string FullName { get; set; }
		string PositionTitle { get; set; }
		string EmailAddress { get; set; }
		string PhoneNumber { get; set; }
		string PicturePath { get; set; }
		int? CountryID { get; set; }
		string CountryName { get; set; }
		int? PostID { get; set; }
		string PostName { get; set; }
		int? DefaultBusinessUnitID { get; set; }
		string DefaultBusinessUnitName { get; set; }
		string DefaultBusinessUnitAcronym { get; set; }

		List<IAppRole_Item> AppRoles { get; set; }
		List<IAppPermission_Item> AppPermissions { get; set; }
		List<IBusinessUnit_Item> BusinessUnits { get; set; }
	}
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UserService.Models/iuser-profile_item.ts



import { IBusinessUnit_Item } from './ibusiness-unit_item';
import { IAppRole_Item } from './iapp-role_item';
import { IAppPermission_Item } from './iapp-permission_item';

export interface IUserProfile_Item {
  
	AppUserID: number;
	ADOID: string;
	First: string;
	Middle: string;
	Last: string;
	FullName: string;
	PositionTitle: string;
	EmailAddress: string;
	PhoneNumber: string;
	PicturePath: string;
	CountryID?: number;
	CountryName: string;
	PostID?: number;
	PostName: string;
	DefaultBusinessUnit?: IBusinessUnit_Item;
	DefaultAppRole?: IAppRole_Item;
	ModifiedByAppUserID: number;
	AppRoles?: IAppRole_Item[];
	AppPermissions?: IAppPermission_Item[];
	BusinessUnits?: IBusinessUnit_Item[];

}

